oidc-controller/api/templates/verified_credentials.html

```diff
@@ -217,45 +210,56 @@ <h1>Scan with a Digital Wallet</h1>
     </div>
   </body>
   <script>
-    let state = "intro"; // intro, success, failure
+    let state = "intro"; // intro, success, pending
 
     const toggleState = (state) => {
       const intro = document.querySelector(".intro");
       const success = document.querySelector(".success");
-      const failure = document.querySelector(".failure");
+      const pending = document.querySelector(".pending");
+      const qrcode = document.querySelector(".qr-code");
       const refreshButton = document.getElementById("refresh-button");
 
       switch (state) {
         case "intro":
           // Header elements
           intro.style.display = "grid";
           success.style.display = "none";
-          failure.style.display = "none";
+          pending.style.display = "none";
 
           // Button elements
           refreshButton.style.display = "none";
+
+          // Turn off the spinner
+          qrcode.classList.remove("pending");
           break;
         case "success":
+          console.log("Success!");
           // Header elements
           intro.style.display = "none";
           success.style.display = "grid";
-          failure.style.display = "none";
+          pending.style.display = "none";
 
           // Button elements
           refreshButton.style.display = "flex";
 
-          setTimeout(() => {
-            window.location.replace("{{callback_url}}", {method: 'POST'})
-          }, 3000);
+          // Turn off the spinner
+          qrcode.classList.remove("pending");
+
+          // setTimeout(() => {
+          //   window.location.replace("{{callback_url}}", {method: 'POST'})
+          // }, 3000);
```

Comment:
This needs to be uncommented, otherwise functionality will be broken (it will never redirect).